tool/teleport/common/sftp.go

```diff
@@ -89,10 +91,34 @@ type allowedOps struct {
 type sftpHandler struct {
 	logger  *log.Entry
 	allowed *allowedOps
-	events  chan<- *apievents.SFTP
+	mtx     sync.Mutex
```

Comment:
It's not really clear now which fields the mutex is meant to protect.

General practice is to group the mutex and the fields it protects to help convey when you need to acquire the mutex.

```
type sftpHandler struct {
    logger *log.Entry
    allowed *allowedOps

     // mtx protects the fields that follow
     mtx sync.Mutex
     files []*trackedFile

     events chan<- apievents.AuditEvent
}
```

Reply:
Done